roles/zuul/files/jobs/hoist.yaml

```diff
@@ -22,3 +23,4 @@
       - shell: |
           ./tests/signed-off-by-test.sh
           python tests/layout-checks.py job-list.txt
+          ./tests/shellcheck-test.sh
```

Comment:
This is a nit to the nth degree, but can we keep tests of the same file extension grouped together like they are in `.travis.yml`?